experiments/axon_geometry/README.md

```diff
@@ -0,0 +1,35 @@
+# Reproduction instructions
+
+Follow these steps to reproduce the results of the experiment:
```

Comment:
Please add a comment instructing users to install torch (I don't think we should add it to package requirements yet)

Reply:
Addressed